v2/cmd/integration-test/integration-test.go

```diff
@@ -73,7 +81,7 @@ func debugTests() {
 			continue
 		}
 		if err := testcase.Execute(tpath); err != nil {
-			panic(err)
+			fmt.Printf("\n%v", err.Error())
```

Comment:
wouldn't this prevent the process from exiting with status different from zero and invalidate GH actions checks?

Reply:
@Mzack9999 , this is only callled when used with `debug.sh` for local debugging and does not relate to gh action .
Earlier I opted for panic but since we have `errorutils` we can just dump stacktrace without panic